src/server/handlers/github.js

```diff
@@ -321,6 +321,19 @@ const getRequest = (owner, name, token, secret) => {
   };
 };
 
+
+export const hasRepoChanged = async (repositoryUrl, since, token) => {
+  if (since === undefined || !since) {
+    throw new Error('`since` must be an ISO8601 value.');
+  }
+  const { owner, name } = parseGitHubRepoUrl(repositoryUrl);
+  const uri = `/repos/${owner}/${name}/commits?since=${since}`;
+  const response = await requestGitHub.get(uri, { token, json: true });
+  await checkGitHubStatus(response);
```

Comment:
This doesn't handle exceeding the rate limit (403). You can check distance from the limit at https://api.github.com/rate_limit.